docker/buildkite/docker-compose-local.yml

```diff
@@ -92,27 +92,13 @@ services:
     build:
       context: ../../
       dockerfile: ./docker/buildkite/Dockerfile
-    command: sh -c "make .just-build && make install-schema && make cover_profile"
+    command: sh -c "make .just-build && make cover_profile"
     environment:
-      - "CASSANDRA_HOST=cassandra"
+      # keep them to be sure that tests requiring dependencies are not skipped
```

Comment:
I'm guessing the intent here is to cause any test depending on databases to fail, so `unit-test` makes sure it never runs those packages at all?

I suppose that'll bake in "integ test dirs should not have any non-integ-tests in them" as a rule.  seems reasonable, though that might be a pain to maintain when we change to `go test ./...`

Reply:
Yeah, just a check to be sure that we will not include any tests there. At the same time, I think we can remove it later, because docker-compose already doesn't have any dependencies so these test will fail. I will remove it in the next PR.